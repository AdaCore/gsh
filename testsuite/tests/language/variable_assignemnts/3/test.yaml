title: Check influence of variable assignment on $?
