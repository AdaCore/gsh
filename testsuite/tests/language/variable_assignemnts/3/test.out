check value of 0 after assignment
a: 10; b: 0; c: 0
a: 11; b: 0; c: 0
a: notset; b: 0; c: 0
a: 00; b: 0; c: 0
a: 1; b: 11; c: 0; d: 0
a: 1; b: ; c: 1; d: 1
